examples/pipeline/3_pipeline_dict_with_services_full.py

```diff
@@ -11,38 +11,31 @@
 from dff.core.engine.core import Context, Actor
 
 from dff.core.pipeline import CLIMessengerInterface, Service, Pipeline, ServiceRuntimeInfo
-from examples.pipeline._pipeline_utils import SCRIPT, get_auto_arg, auto_run_pipeline
+from _pipeline_utils import SCRIPT, should_auto_execute, auto_run_pipeline
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 """
```

Comment:
Do we need this documentation fixes? pls, look throw all  code

Reply:
Yes, it is required for documentation building.
In a word: during documentation building we create soft links under `docs/sources` and execute example files there. There is no `example.pipeline` module in there, so we face an import error.